BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -57,6 +77,12 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
         notifyTextChanged = { [self] in
             guard urlTextField.isEditing else { return }
 
+            if urlTextField.text?.isEmpty == true {
+                hideClearButton()
+            } else if urlTextField.text?.count == 1 {
```

Comment:
`>= 1` instead of `== 1`?

Reply:
This approach prevents the code from being called multiple times. If we used `>=`, the code would execute each time a character is typed.

Reply:
Then what about the user pasting something? In that case the clear button wouldn't be shown.